pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -379,18 +390,36 @@ func (a *ofLearnAction) MatchLearnedTCPDstPort() LearnAction {
 	return a.MatchTransportDst(ProtocolTCP)
 }
 
+// MatchLearnedTCPv6DstPort specifies that the tcp_dst field in the learned flow
+// must match the tcp_dst of the packet currently being processed.
+func (a *ofLearnAction) MatchLearnedTCPv6DstPort() LearnAction {
+	return a.MatchTransportDst(ProtocolTCPv6)
+}
+
 // MatchLearnedUDPDstPort specifies that the udp_dst field in the learned flow
 // must match the udp_dst of the packet currently being processed.
 func (a *ofLearnAction) MatchLearnedUDPDstPort() LearnAction {
 	return a.MatchTransportDst(ProtocolUDP)
 }
 
-// MatchLearnedUDPDstPort specifies that the sctp_dst field in the learned flow
+// MatchLearnedUDPv6DstPort specifies that the udp_dst field in the learned flow
+// must match the udp_dst of the packet currently being processed.
+func (a *ofLearnAction) MatchLearnedUDPv6DstPort() LearnAction {
+	return a.MatchTransportDst(ProtocolUDPv6)
```

Comment:
Please take care of the implementation of function `MatchTransportDst` (https://github.com/vmware-tanzu/antrea/blob/master/pkg/ovs/openflow/ofctrl_action.go#L363), I remembered there should be some hardcode for IP protocol. If you plan to use the same function, you need to ensure that function is also available for IPv6 protocols.

Reply:
Done